source/Calamari/Scripts/Octopus.Features.IISWebSite_BeforePostDeploy.ps1

```diff
@@ -467,12 +467,15 @@ if ($deployAsWebSite)
 		# in the deployment process
 		if ($_.certificateVariable) {
 			$sslCertificateThumbprint = $OctopusParameters[$_.certificateVariable + ".Thumbprint"]
-		} else {
+		} elseif ($_.thumbprint){
 			# Otherwise, the certificate thumbprint was supplied directly in the binding
 			$sslCertificateThumbprint = $_.thumbprint.Trim()
+		} else {
+			[Console]::Error.WriteLine("Cannot configure a https binding when neither certificate thumbprint nor certificate variable is not supplied on the binding.")
```

Comment:
I got confused for the first 3 attempts at reading this. :)

```suggestion
			Write-Error "To configure an HTTPS binding please choose a certificate which is managed by Octopus, or provide the thumbprint for a certificate which is already available in the Machine certificate store."
```

Reply:
Yeah - i had trouble coming up with the words.
I was using `Error.WriteLine(...)` as the `Write-Error` was being truncated and wasn't readable.